### PR TITLE
[zh-CN] sync Notification constructor

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

sync Notification constructor

### Motivation

sync Notification constructor

### Additional details

https://github.com/mdn/content/blob/main/files/en-us/web/api/notification/notification/index.md?plain=1

https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification

### CODE REVIEW

**Comment by yin1999** [comment]
Hi @skyclouds2001. Do you need any help on this :)

**Comment by skyclouds2001** [comment]
> Hi @skyclouds2001. Do you need any help on this :)

Oppps，I forgot this.

**Comment by skyclouds2001** [comment]
I'll work to finish this.

**Comment by skyclouds2001** [comment]
Very Sorry for the delay

File: files/zh-cn/web/api/notification/notification/index.md [hunk 1]
```diff
@@ -1,58 +1,76 @@
...
+- `title`
+  - : 定义一个通知的标题，它将显示在通知窗口的顶部。
+- `options` {{optional_inline}}
+
+  - : 一个包含要应用于通知的任何自定义设置的选项对象。可能的选项有：
+
+    - `dir`
+      - : 显示通知的方向。它默认为 `auto`，即只采用浏览器的语言设置行为，但您可以通过设置 `ltr` 和 `rtl` 的值来覆盖该行为（尽管大多数浏览器似乎忽略这些设置）。
+    - `lang`
+      - : 指定通知的语言，根据 {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}} 使用表示语言标签的字符串指定。请参阅 Sitepoint [ISO 2 字母语言代码](https://www.sitepoint.com/iso-2-letter-language-codes/) 页面以获取简单参考。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:28]:
```suggestion
      - : 指定通知的语言，根据{{RFC(5646, "识别语言的标签（也被称为 BCP47）")}}使用表示语言标签的字符串指定。请参阅 Sitepoint [ISO 2 字母语言代码](https://www.sitepoint.com/iso-2-letter-language-codes/)页面以获取简单参考。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 2]
```diff
@@ -1,58 +1,76 @@
...
+    - `icon`
+      - : 一个包含要在通知中显示的图标的 URL 的字符串。
+    - `image`
+      - : 一个包含要在通知中显示的图像的 URL 的字符串。
+    - `data`
+      - : 任意你想要与通知关联的数据。它可以是任何数据类型。
+    - `vibrate`
+      - : 设备振动硬件随通知一起发出的[振动模式](/zh-CN/docs/Web/API/Vibration_API#振动模式)。当该值被指定时，`silent` 参数不得设置为 `true`。
+    - `renotify`
+      - : 一个布尔值，指定在新通知替换旧通知后是否应通知用户。默认值为 `false`，这意味着他们不会收到通知。如果该值被指定为 `true`，那么 `tag` 参数必须被设置。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:44]:
```suggestion
      - : 一个布尔值，指定在新通知替换旧通知后是否应通知用户。默认值为 `false`，这意味着他们不会收到通知。如果该值被指定为 `true`，那么必须同时设置 `tag` 参数。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 3]
```diff
@@ -1,58 +1,76 @@
...
+    - `requireInteraction`
+      - : 指示通知应保持活动状态，直到用户单击或关闭它，而不是自动关闭。默认值为 `false`。
+    - `actions`
+
+      - : 要在通知中显示的一系列操作的数组。数组中的每个元素都是一个具有以下成员的对象：
+
+        - `action`: 一个标识要在通知上显示的用户操作的字符串。
+        - `title`: 一个包含要向用户显示的操作文本的字符串。
+        - `icon`: 一个包含与操作一起显示的图标 URL 的字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:53]:
```suggestion
        - `action`：一个标识要在通知上显示的用户操作的字符串。
        - `title`：一个包含要向用户显示的操作文本的字符串。
        - `icon`：一个包含与操作一起显示的图标 URL 的字符串。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 4]
```diff
@@ -1,58 +1,76 @@
...
+
+        使用 {{domxref("ServiceWorkerGlobalScope.notificationclick_event", "notificationclick")}} 事件中的 `event.action` 构建适当的响应。
+
+    - `silent`
+      - : 一个布尔值，指定通知是否静音（不发出声音或振动），无论设备设置如何。默认值为 `false`，这意味着它不会保持静音。如果被设置为 `true`，那么 `vibrate` 参数不能被传递。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:58]:
```suggestion
      - : 一个布尔值，指定通知是否静音（不发出声音或振动），无论设备设置如何。默认值为 `false`，这意味着它不会保持静音。如果被设置为 `true`，那么不能同时传递 `vibrate` 参数。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 5]
```diff
@@ -1,58 +1,76 @@
...
+
+### 异常
+
+- {{jsxref("TypeError")}}
+  - : 如果满足以下条件之一，则抛出该异常：
+    - `options.silent` 是 `true` 并且 `options.vibrate` 被指定。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:64]:
```suggestion
    - `options.silent` 是 `true` 且同时指定了 `options.vibrate`。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 6]
```diff
@@ -1,58 +1,76 @@
...
+    - `options.renotify` 是 `true` 但是 `options.tag` 为空字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:65]:
```suggestion
    - `options.renotify` 是 `true` 但是 `options.tag` 为空值。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 7]
```diff
@@ -1,58 +1,76 @@
...
 
 ## 示例
 
-In our [Emogotchi demo](http://mdn.github.io/emogotchi/) ([see source code](https://github.com/mdn/emogotchi)), we run a simple `spawnNotification()` function when we want to fire a notification — this is passed arguments to specify the body, icon and title we want, then it creates the necessary `options` object and fires the notification using the `Notification()` constructor.
+在我们的 [`Emogotchi 演示`](https://chrisdavidmills.github.io/emogotchi/)（[查看源代码](https://github.com/chrisdavidmills/emogotchi)）中，当我们想要触发通知时，我们运行 `spawnNotification()` 函数。该函数传递参数来指定我们想要的正文、图标和标题，然后它创建必要的 `options` 对象并使用 `Notification()` 构造函数触发通知。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:69]:
```suggestion
在我们的 [`Emogotchi` 演示](https://chrisdavidmills.github.io/emogotchi/)（[查看源代码](https://github.com/chrisdavidmills/emogotchi)）中，当我们想要触发通知时，我们运行 `spawnNotification()` 函数。该函数传递参数来指定我们想要的正文、图标和标题，然后它创建必要的 `options` 对象并使用 `Notification()` 构造函数触发通知。
```

File: files/zh-cn/web/api/notification/notification/index.md [hunk 8]
```diff
@@ -64,6 +82,12 @@ function spawnNotification(theBody, theIcon, theTitle) {
 
 {{Compat}}
 
+### Chrome 浏览器注意点
+
+从 Chrome 49 开始，通知在无痕模式下不起作用。
+
+Android 版 Chrome 会在调用 `Notification` 构造函数时抛出一个 {{jsxref("TypeError")}}。它仅支持从 Service Worker 中创建通知。有关更多详细信息，请参阅 [Chromium 问题跟踪器](https://crbug.com/481856)。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/notification/index.md:89]:
```suggestion
### Chrome 浏览器备注

从 Chrome 49 开始，通知在无痕模式下不起作用。

Android 版 Chrome 会在调用 `Notification` 构造函数时抛出 {{jsxref("TypeError")}}。它仅支持从 Service Worker 中创建通知。有关更多详细信息，请参阅 [Chromium 问题跟踪器](https://crbug.com/481856)。
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions

**Comment by yin1999** [comment]
> Very Sorry for the delay

Don't worry about this. Thanks for the contribution :)

**Comment by yin1999** [comment]
The better way to apply multi-suggestions is go to [files changed](https://github.com/mdn/translated-content/pull/14075/files) menu, and then [apply suggestions in batches](https://docs.github.com/zh/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#:~:text=%E5%8D%95%E5%87%BB%E2%80%9C%E6%8F%90%E4%BA%A4%E5%BB%BA%E8%AE%AE%E2%80%9D%E3%80%82-,%E8%8B%A5%E8%A6%81%E5%B0%86%E5%BB%BA%E8%AE%AE%E6%B7%BB%E5%8A%A0%E5%88%B0%E4%B8%80%E6%89%B9%E6%9B%B4%E6%94%B9,-%EF%BC%8C%E8%AF%B7%E5%8D%95%E5%87%BB%E2%80%9C%E6%B7%BB%E5%8A%A0).

**Review by yin1999 — APPROVED** [review]
Thanks @skyclouds2001

**Pull request merged**